src/lib/plugins/blocks.js

```diff
@@ -4,12 +4,7 @@ module.exports.player=function(player,serv)
 {
   player.changeBlock=async (position,blockType,blockData) =>
   {
-    serv.players
-      .filter(p => p.world==player.world && player!=p)
```

Comment:
indeed the vanilla server doesn't echo it. Except if you (the client) do something wrong